test_demo.py

```diff
@@ -8,10 +8,10 @@ def test_show_aggie_pride(self):
         df = show_aggie_pride()
         self.assertEqual(df.loc[0, 'Text'], 'Aggie Pride')
         self.assertEqual(df.loc[1, 'Text'], 'Worldwide')
-        self.assertEqual(df.loc[2, 'Text'], 'Hard Work Yields Results')        
-        self.assertEqual(df.loc[3, 'Text'], 'Work today for what you want to achieve tomorrow')
+        self.assertEqual(df.loc[2, 'Text'], 'Hard Work Yields Results')
+	self.assertEqual(df.loc[3, 'Text'], 'Work today for what you want to achieve tomorrow')
```

Comment:
This should be changed to spaces instead of tab to match the rest of the file.  That's what's causing [this](https://github.com/NCATComp410/comp410_summer21/pull/9/checks?check_run_id=2871348871) error